Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Beam/Volunteer.hs

```diff
@@ -0,0 +1,47 @@
+{-
+  Copyright 2022-23, Juspay India Pvt Ltd
+
+  This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+  as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+  is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+  the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE DerivingStrategies #-}
+
+module Storage.Beam.Volunteer where
+
+import Data.Serialize
+import qualified Data.Time as Time
+import qualified Database.Beam as B
+import Database.Beam.MySQL ()
+import EulerHS.KVConnector.Types (KVConnector (..), MeshMeta (..), primaryKey, secondaryKeys, tableName)
+import GHC.Generics (Generic)
+import Kernel.Prelude hiding (Generic)
+import Lib.Utils ()
+import Sequelize
+import Tools.Beam.UtilsTH
+
+data VolunteerT f = VolunteerT
```

Comment:
Don't we need to add this new table to drainer?

Reply:
added